addons/website/static/src/client_actions/website_preview/website_preview.xml

```diff
@@ -13,7 +13,8 @@
          t-att-class="{ 'editor_enable editor_has_snippets': this.websiteContext.snippetsLoaded,
                         'o_is_blocked': this.blockedState.isBlocked,
                         'o_is_mobile': this.websiteContext.isMobile,
-                        'border-top': !this.websiteContext.snippetsLoaded }">
+                        'border-top': !this.websiteContext.snippetsLoaded,
+                        'o_is_microsoft_edge': isMicrosoftEdge }">
```

Comment:
Instead of having to break the history of the line above each time we'll add a class to this list, move the `}">` to the next line (and add a trailing comma before). I would probably align the `}` with the above `{`.